src/main/kotlin/no/nav/personbruker/dittnav/eventaggregator/common/database/Database.kt

```diff
@@ -51,4 +49,13 @@ interface Database {
         }
     }
 
+    fun queryWithExceptionTranslation(operationToExecute: Connection.() -> Unit) {
+        translateExternalExceptionsToInternalOnes {
+            runBlocking {
```

Comment:
Burde vi ha trukket ut `runBlocking` her fra, og heller tatt stilling til om man trenger det der man kaller `queryWithExceptionTranslation`? Hva er egentlig best practices her?

Reply:
Skal undersøke litt. Flyttet det i hovedsak ned hit for å rydde opp i innrykk siden translateExceptions funksjonen alltid ble kalt med runblocking likevel.